<?xml version="1.0"?>
<launch>

    <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>

    <!-- Device Argument -->
    <arg name="Frame_ID" default="greenhouse"/>
    <arg name="MAC_addr" default="80:EA:CA:88:D7:E2"/>
    <!-- Sample Rate miniumum is once per minute 0.01667 hz -->
    <arg name="sample_rate" default="0.01667"/>

    <!-- Client Node -->
    <node pkg="miflora_server" type="sensor_client.py" name="sensor_client" clear_params="true" output="screen" >
      <param name="Frame_ID" value="$(arg Frame_ID)" />
      <param name="MAC_addr" value="$(arg MAC_addr)" />
      <param name="sample_rate" value="$(arg sample_rate)" />
    </node>

</launch>